conf/metrics.properties.template

```diff
@@ -1,7 +1,6 @@
 # Enable StatsdSink for all instances by class name
 *.sink.statsd.class=org.apache.spark.metrics.sink.statsd.StatsdSink
-*.sink.statsd.prefix=spark
-*.sink.statsd.tags=app_type=spark
```

Comment:
let's keep `*.sink.statsd.tags=` without a value for documentation/example purposes

Reply:
changed as suggested